Cargo.toml

```diff
@@ -16,17 +16,20 @@ edition = "2021"
 license = "MIT OR Apache-2.0"
 
 [dependencies]
-bytemuck = "1.7.3"
-byteorder = "1.4.3"
-serde = { version = "1.0.139", optional = true }
+bytemuck = { version = "1.7", optional = true }
+byteorder = { version = "1.4", optional = true }
+serde = { version = "1.0", optional = true }
 
 [features]
+default = ["std"]
+serde = ["dep:serde", "std"]
 simd = []
+std = ["dep:bytemuck", "dep:byteorder"]
 
 [dev-dependencies]
-proptest = "1.2.0"
-serde_json = "1.0.85"
-bincode = "1.3.3"
+proptest = "1.2"
+serde_json = "1.0"
+bincode = "1.3"
```

Comment:
Please don't remove patch version numbers. I know the patch version is not the lock file. However, as this is a library, there are no lock files. We can rely on this to know which version we are correctly compiling if we have an issue at some point.

Reply:
sure, lgtm.